tests/client/test_client_task_meilisearch.py

```diff
@@ -151,7 +151,7 @@ def test_delete_tasks_by_uid(client, empty_index, small_movies):
     assert f"uids={task_addition.task_uid}" in task["details"]["originalFilter"]
 
 
-def test_delete_all_tasks(client):
+def test_delete_tasks_by_filter(client):
```

Comment:
Yes, I have updated my PR comment.

Reply:
Like @sanders41, the change on the test file should not be here
First, because it is not a good practice to solve multiple issues in the same PR, especially when they are simple problems. Second, changing the name of the tests does not solve the issue.

Reply:
Sure, I have modified accordingly.